src/FrameManager.ts

```diff
@@ -73,7 +70,7 @@ class FrameManager {
    */
   public sendToClient(message: HostToClient) {
     const clientOrigin = this._expectedClientOrigin();
-    if (this._iframe.contentWindow) {
+    if (this._iframe.contentWindow && clientOrigin) {
       const validated = validateIncoming(message);
```

Comment:
Oops. This is a mistake on my part. `validateIncoming` here should be `validateOutgoing`.  I just got lucky that right now the `HostToClient` is a subset of `ClientToHost`